test/JDBC/src/test/java/com/sqlsamples/TestQueryFile.java

```diff
@@ -424,6 +425,7 @@ public void TestQueryBatch(String inputFileName) throws SQLException, ClassNotFo
         BufferedWriter bw = new BufferedWriter(fw);
         curr_exec_time = 0L;
         checkParallelQueryExpected = false;
+        checkSingleDbModeExpected = false;
```

Comment:
I think rather than having this flag in test file, better to check whether there is any expected file for single_db mode or not just like we did for non-default server collation.
Because we want to run single-db mode upgrade tests as well, if we follow this approach..i think it will be more difficult to do that

Reply:
Can you explain how it will be difficult to run upgrade in single-db mode with this approach? I think searching of file only if required (i.e. flag is specified) is better approach than always searching for file in folder.